Sources/ClientGtk/ConfigManager.swift

```diff
@@ -0,0 +1,9 @@
+import DeltaCore
+
+/// Dummy implementation of ClientConfiguration as required by DeltaCore.
+class CoreConfiguration: ClientConfiguration {
+	public var render = RenderConfiguration()
+	public var keymap = Keymap.default
+	public var toggleSprint = false
+	public var toggleSneak = false
+}
```

Comment:
Add a trailing newline to this file (it's just a convention thing, but GitHub puts a scary symbol so I do it)